src/gatsby/src/components/search/filter.js

```diff
@@ -59,13 +58,13 @@ const Filter = ({ filterItems, selectedFilterItems, handleFilterCheck, filterTyp
               checked={checked}
               onChange={event => {
                 handleFilterCheck(item, event)
-                trackStructEvent({
-                  category: 'Checkbox',
-                  action: event.target.checked ? 'Check' : 'Uncheck',
-                  label: item.label,
-                  property: 'filter-checkbox',
-                  value: event.target.checked ? 1 : 0
-                })
+                window.snowplow(
+                  "trackStructEvent",
+                  "Checkbox",
+                  event.target.checked ? "Check" : "Uncheck",
+                  item.label, "filter-checkbox",
```

Comment:
In addition to the label, it would be good to track the category of the checkbox. e.g. "Popular" / "Area" / Camping / "Activities" / "Facilities"